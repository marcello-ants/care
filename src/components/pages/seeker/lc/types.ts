import { GeneralProviderProfile } from '@/types/seeker';

export interface ProviderProfile extends GeneralProviderProfile {
  numberOfReviews: number;
  qualities?: ProviderQualities;
  services?: ProviderServices;
  firstName: string;
}

export interface ProviderQualities {
  certifiedNursingAssistant?: boolean;
  certifiedRegisterNurse?: boolean;
  certifiedTeacher?: boolean;
  childDevelopmentAssociate?: boolean;
  comfortableWithPets?: boolean;
  cprTrained?: boolean;
  crn?: boolean;
  doesNotSmoke?: boolean;
  doula?: boolean;
  earlyChildDevelopmentCoursework?: boolean;
  earlyChildhoodEducation?: boolean;
  firstAidTraining?: boolean;
  nafccCertified?: boolean;
  ownTransportation?: boolean;
  specialNeedsCare?: boolean;
  trustlineCertifiedCalifornia?: boolean;
}

export interface ProviderServices {
  carpooling?: boolean;
  craftAssistance?: boolean;
  errands?: boolean;
  groceryShopping?: boolean;
  laundryAssistance?: boolean;
  lightHousekeeping?: boolean;
  mealPreparation?: boolean;
  swimmingSupervision?: boolean;
  travel?: boolean;
}
