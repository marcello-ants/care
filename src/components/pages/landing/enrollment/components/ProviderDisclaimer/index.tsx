// External Dependencies
import { Typography, Link } from '@care/react-component-lib';

// Styles
import useStyles from './styles';

export default function ProviderDisclaimer() {
  // Styles
  const classes = useStyles();

  return (
    <Typography careVariant="disclaimer1" className={classes.disclaimer}>
      <span>
        You must be 18 years or older to use Care.com. By clicking &quot;Submit,&quot; you agree to
        our
      </span>{' '}
      <Link href="/terms-of-use-p1012.html" careVariant="disclaimer1">
        Terms of Use
      </Link>{' '}
      <span>and</span>{' '}
      <Link href="/privacy-policy-p1013.html" careVariant="disclaimer1">
        Privacy Policy
      </Link>
      <span>.</span>
    </Typography>
  );
}
