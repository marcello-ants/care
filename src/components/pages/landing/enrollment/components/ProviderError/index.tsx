// External Dependencies
import { Box } from '@material-ui/core';
import { Icon24UtilityWarning } from '@care/react-icons';
import { Typography } from '@care/react-component-lib';

// Styles
import useStyles from './styles';

function ProviderError() {
  // Styles
  const classes = useStyles();

  return (
    <Box className={classes.errorBanner}>
      <Box>
        <Icon24UtilityWarning color="#CF230C" />
      </Box>
      <Box>
        <Typography variant="h4">Whoops! Something went wrong</Typography>
        <Typography careVariant="body3">
          There was an error when we tried to submit your request. Please try again.
        </Typography>
      </Box>
    </Box>
  );
}

export default ProviderError;
