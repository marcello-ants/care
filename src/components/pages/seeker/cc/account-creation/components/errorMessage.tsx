import React from 'react';
import { Box, Grid } from '@material-ui/core';
import { Alert } from '@material-ui/lab';

type ErrorMessageData = {
  errorMessageText: string;
};

const ErrorMessageNotification = (props: ErrorMessageData) => {
  const { errorMessageText } = props;

  return (
    <Grid item xs={12}>
      <Box mb={3}>
        <Alert severity="error">{errorMessageText}</Alert>
      </Box>
    </Grid>
  );
};

export default ErrorMessageNotification;
