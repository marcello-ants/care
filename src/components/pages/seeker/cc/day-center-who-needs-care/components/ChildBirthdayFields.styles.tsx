// External Dependencies
import { makeStyles } from '@material-ui/core';
import { theme } from '@care/material-ui-theme';

const useStyles = makeStyles({
  childContainer: {
    padding: theme.spacing(0),
  },
  childContainerNth: {
    marginTop: theme.spacing(4),
    position: 'relative',
  },
  optionsContainer: {
    display: 'flex',
    flexDirection: 'row',
  },
  fullWidth: {
    width: '100%',
  },
  error: {
    color: theme.palette.care.red[800],
    paddingLeft: theme.spacing(1),
  },
  selectContainer: {
    marginTop: theme.spacing(1),
  },
  monthSelect: {
    marginRight: theme.spacing(0.5),
    '& label': {
      paddingLeft: theme.spacing(1),
    },
  },
  yearSelect: {
    marginLeft: theme.spacing(0.5),
    '& label': {
      paddingLeft: theme.spacing(1),
    },
  },
  removeChildButton: {
    position: 'absolute',
    top: '0',
    right: '0',
    width: 'auto',
    height: 'auto',
    lineHeight: 'unset',
    minWidth: 'auto',
    padding: theme.spacing(0),
    color: theme.palette.care.blue[700],
    '&:hover': {
      backgroundColor: 'transparent',
    },
  },
});

export default useStyles;
