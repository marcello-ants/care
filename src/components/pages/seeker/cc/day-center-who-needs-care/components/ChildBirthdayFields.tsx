// External Dependencies
import React, { useMemo } from 'react';
import { Button, FormHelperText, Grid, InputLabel, MenuItem } from '@material-ui/core';
import { Select, Typography } from '@care/react-component-lib';

// Internal Dependencies
import { ChildDateOfBirth } from '@/types/seekerCC';
import { getBirthAvailableMonthsAndYears } from '../utils';
import useStyles from './ChildBirthdayFields.styles';

// Interfaces
interface TErrorProps {
  year?: string;
  month?: string;
}

type SelectInputProps = React.ComponentProps<typeof Select>;
interface IChildBirthdayFieldsProps {
  child: ChildDateOfBirth;
  onRemoveChild: () => void;
  onChange: SelectInputProps['onChange'];
  childIndex: number;
  error?: TErrorProps;
  allow9MonthsOutFromNow?: boolean;
}

function ChildBirthdayFields({
  child,
  childIndex: i,
  onChange,
  onRemoveChild,
  error,
  allow9MonthsOutFromNow,
}: IChildBirthdayFieldsProps) {
  const classes = useStyles();

  const { months, years } = useMemo(
    () => getBirthAvailableMonthsAndYears(child.month, child.year, allow9MonthsOutFromNow),
    [child.year, child.month, allow9MonthsOutFromNow]
  );

  const monthsOptions = useMemo(() => {
    return months.map((month) => {
      return (
        <MenuItem key={month.value} value={month.value}>
          {month.name}
        </MenuItem>
      );
    });
  }, [months]);

  const yearsOptions = useMemo(() => {
    return years.map((year: string) => {
      return (
        <MenuItem key={year} value={year}>
          {year}
        </MenuItem>
      );
    });
  }, [years]);

  return (
    <Grid
      // eslint-disable-next-line react/no-array-index-key
      item
      xs={12}
      className={`${
        i === 0
          ? `${classes.childContainer}`
          : `${classes.childContainer} ${classes.childContainerNth}`
      }`}>
      <Typography variant="h3">Child {i + 1}</Typography>
      {i !== 0 && (
        <Button className={classes.removeChildButton} onClick={onRemoveChild}>
          Remove Child
        </Button>
      )}
      <Grid item xs={12} className={classes.optionsContainer}>
        <Grid item xs={6} className={`${classes.selectContainer} ${classes.monthSelect}`}>
          <InputLabel id={`month-select-label-${i}`}>Birth month</InputLabel>
          <Select
            className={classes.fullWidth}
            labelId={`month-select-label-${i}`}
            id={`month-select-child-${i}`}
            name={`children[${i}].month`}
            onChange={onChange}
            value={child.month}
            error={Boolean(error?.month)}>
            {monthsOptions}
          </Select>
          {error?.month && (
            <FormHelperText className={classes.error}>Select their birth month</FormHelperText>
          )}
        </Grid>
        <Grid item xs={6} className={`${classes.selectContainer} ${classes.yearSelect}`}>
          <InputLabel id={`year-select-label-${i}`}>Birth year</InputLabel>
          <Select
            className={classes.fullWidth}
            labelId={`year-select-label-${i}`}
            id={`year-select-child-${i}`}
            name={`children[${i}].year`}
            onChange={onChange}
            value={child.year}
            error={Boolean(error?.year)}>
            {yearsOptions}
          </Select>
          {error?.year && (
            <FormHelperText className={classes.error}>Select their birth year</FormHelperText>
          )}
        </Grid>
      </Grid>
    </Grid>
  );
}

ChildBirthdayFields.defaultProps = {
  error: undefined,
  allow9MonthsOutFromNow: undefined,
};

export type { IChildBirthdayFieldsProps, TErrorProps };
export default ChildBirthdayFields;
