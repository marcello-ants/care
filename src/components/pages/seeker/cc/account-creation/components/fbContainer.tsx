import { Box, makeStyles, Grid, Button } from '@material-ui/core';
import { useFacebookApiKey } from '@/components/FacebookApiKeyContext';

// @ts-ignore
import FacebookLogin from 'react-facebook-login/dist/facebook-login-render-props';
import { Icon48SocialMediaFacebook } from '@care/react-icons';

export type facebookResponseProps = {
  accessToken: string;
  // eslint-disable-next-line
  data_access_expiration_time?: number;
  email: string;
  expiresIn?: number;
  graphDomain?: string;
  id?: string;
  name: string;
  signedRequest?: string;
  userID?: string;
};

export interface SplitFormValues {
  email: string;
}

export interface SplitFormWithFbData extends SplitFormValues {
  firstName?: string;
  lastName?: string;
  accessToken?: string;
}

const mobileWidth = 360;
const fbColor = '#1877F2';
const fbIconSize = '35px';
const useStyles = (fbEmphasizedDesign: boolean) =>
  makeStyles((theme) => ({
    fbButton: {
      paddingTop: theme.spacing(5),
      borderColor: fbEmphasizedDesign ? fbColor : theme.palette.default?.dark,
      background: fbEmphasizedDesign ? fbColor : theme.palette.care?.white,
      color: fbEmphasizedDesign ? theme.palette.care?.white : theme.palette.default?.dark,
      fontSize: fbEmphasizedDesign ? '21px' : '18px',
      '&:hover': {
        background: fbEmphasizedDesign ? fbColor : theme.palette.care?.white,
      },
    },
    fbButtonContent: {
      display: 'grid',
      gridTemplateColumns: 'repeat(3, minmax(0, 1fr))',
      [theme.breakpoints.down(mobileWidth)]: {
        display: 'flex',
        alignItems: 'center',
      },
    },
    fbIconContainer: {
      display: 'flex',
      alignItems: 'center',
      justifyContent: 'center',
      width: fbIconSize,
      '& svg': {
        width: fbIconSize,
        height: fbIconSize,
      },
    },
    fbIcon: {
      background: '#1a77f2',
      borderRadius: '19px',
    },
    fbButtonText: {
      justifySelf: 'center',
      marginLeft: '10px',
      marginRight: '5px',
      fontColor: theme.palette.care?.white,
    },
  }));

type GetFbContainerProps = {
  handleFacebookLogin: (response: facebookResponseProps) => Promise<void>;
  fbButtonOnTop: boolean;
  fbButtonOnBottom: boolean;
};

const getFbContainer = ({
  handleFacebookLogin,
  fbButtonOnTop,
  fbButtonOnBottom,
}: GetFbContainerProps) => {
  const fbEmphasizedDesign = fbButtonOnBottom || fbButtonOnTop;
  const classes = useStyles(fbEmphasizedDesign)();
  const { apiKey } = useFacebookApiKey();

  return (
    <Grid item xs={12}>
      <Box pt={fbButtonOnTop ? 0 : 2}>
        <FacebookLogin
          appId={apiKey}
          autoLoad={false}
          fields="name,email"
          callback={handleFacebookLogin}
          render={(renderProps: any) => (
            <Button
              onClick={renderProps.onClick}
              className={classes.fbButton}
              variant="outlined"
              size="large"
              fullWidth>
              <div className={classes.fbButtonContent}>
                <div className={classes.fbIconContainer}>
                  <Icon48SocialMediaFacebook size="38px" color="white" className={classes.fbIcon} />
                </div>
                <p className={classes.fbButtonText}>Continue with Facebook</p>
              </div>
            </Button>
          )}
        />
      </Box>
    </Grid>
  );
};

export default getFbContainer;
