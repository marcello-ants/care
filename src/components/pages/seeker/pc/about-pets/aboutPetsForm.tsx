import { useFormik, FormikErrors, useField } from 'formik';
import { Select, FormControl, InputLabel, MenuItem } from '@material-ui/core';
import { useAppDispatch, useSeekerPCState } from '@/components/AppState';
import useFlowNavigation from '@/components/features/flowNavigation/useFlowNavigation';
import { logCareEvent } from '@/utilities/amplitudeAnalyticsHelper';

export const GENERAL_ERROR_MESSAGE = 'Please select a valid number of pets';

type FormikSelectFieldProps = {
  name: string;
  label: string;
  errorDisplayed: boolean;
  options: number[] | string[];
};

type FormValues = {
  dogs: number;
  cats: number;
  other: number;
  generalFormError: string;
};

type FormErrors = {
  generalFormError: string;
};

export const FormikSelectField = (props: FormikSelectFieldProps) => {
  const { name, label, errorDisplayed, options, ...rest } = props;
  const [field] = useField({ name });
  return (
    <>
      <FormControl fullWidth margin="dense" error={errorDisplayed}>
        <InputLabel htmlFor={name}>{label}</InputLabel>
        <Select data-testid="select" autoWidth {...field} {...rest}>
          {options.map((option: number | string) => (
            <MenuItem key={option} data-testid="select-option" value={option}>
              {option}
            </MenuItem>
          ))}
        </Select>
      </FormControl>
    </>
  );
};

function AboutPetsForm() {
  const { goNext } = useFlowNavigation();
  const dispatch = useAppDispatch();
  const { pets } = useSeekerPCState();

  function handleSubmission(values: FormValues) {
    const { dogs, cats, other } = values;
    dispatch({ type: 'setPetsNumber', pets: { dogs, cats, other } });
    logCareEvent('Member Enrolled', 'Details', { pets: { dogs, cats, other } });
    goNext();
  }

  function formikValidation(values: FormValues) {
    const errors: FormikErrors<FormErrors> = {};
    const atLeastOneFieldFullfiled = Object.values(values).some((value) => Boolean(value));

    if (!atLeastOneFieldFullfiled) {
      errors.generalFormError = GENERAL_ERROR_MESSAGE;
    }
    return errors;
  }

  const formik = useFormik<FormValues>({
    initialValues: {
      ...pets,
      generalFormError: '',
    },
    validate: formikValidation,
    onSubmit: handleSubmission,
    validateOnBlur: false,
    validateOnChange: true,
  });

  return formik;
}

export default AboutPetsForm;
