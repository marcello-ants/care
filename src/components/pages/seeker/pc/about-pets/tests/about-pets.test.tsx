// External Dependencies
import React from 'react';
import { render, screen } from '@testing-library/react';
import { NextRouter, useRouter } from 'next/router';
import { ThemeProvider } from '@material-ui/core';
import { theme } from '@care/material-ui-theme';

import { AppStateProvider } from '@/components/AppState';
import { AppState } from '@/types/app';
import { initialAppState } from '@/state';

import AboutPets from '../about-pets';
import { GENERAL_ERROR_MESSAGE } from '../aboutPetsForm';

jest.mock('next/router', () => ({
  __esModule: true,
  useRouter: jest.fn(),
}));

let mockRouter: Pick<NextRouter, 'push' | 'asPath' | 'pathname'>;

const initialState: AppState = {
  ...initialAppState,
};

function renderPage(state?: AppState) {
  const pathname = '/seeker/pc/about-pets';
  mockRouter = {
    push: jest.fn(),
    asPath: pathname,
    pathname,
  };
  (useRouter as jest.Mock).mockReturnValue(mockRouter);

  return render(
    <ThemeProvider theme={theme}>
      <AppStateProvider initialStateOverride={state || initialState}>
        <AboutPets />
      </AppStateProvider>
    </ThemeProvider>
  );
}

describe('Seeker - Pet Care - About Pets', () => {
  it('matches snapshot', async () => {
    const view = renderPage();

    expect(view.asFragment()).toMatchSnapshot();
  });

  it('shows error if total pets number is equal to 0', async () => {
    renderPage();

    const nextButton = screen.getByRole('button', { name: 'Next' });
    nextButton.click();
    const error = await screen.findByText(GENERAL_ERROR_MESSAGE);
    expect(error).toBeInTheDocument();
  });

  it('shows no error if total pets number is greater than 0', () => {
    const state: AppState = {
      ...initialAppState,
      seekerPC: {
        ...initialAppState.seekerPC,
        pets: { dogs: 1, cats: 0, other: 0 },
      },
    };
    renderPage(state);

    const nextButton = screen.getByRole('button', { name: 'Next' });
    nextButton.click();
    const error = screen.queryByText(GENERAL_ERROR_MESSAGE);
    expect(error).not.toBeInTheDocument();
  });
});
