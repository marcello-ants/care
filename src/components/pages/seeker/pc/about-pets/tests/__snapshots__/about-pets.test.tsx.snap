// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seeker - Pet Care - About Pets matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container"
  >
    <div
      class="MuiGrid-root makeStyles-header-1 MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h2
        class="MuiTypography-root makeStyles-header-1 MuiTypography-h2 MuiTypography-colorTextPrimary"
      >
        Tell us about your pets!
      </h2>
    </div>
    <div
      class="MuiGrid-root makeStyles-subheader-2 MuiGrid-item MuiGrid-grid-xs-12"
    >
      <h3
        class="MuiTypography-root makeStyles-subheader-2 MuiTypography-h3 MuiTypography-colorTextPrimary"
      >
        How many pets need care?
      </h3>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        class="MuiGrid-root makeStyles-selectorContainer-3 MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          class="MuiGrid-root makeStyles-selector-4 MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            class="MuiFormControl-root MuiFormControl-marginDense MuiFormControl-fullWidth"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiFormLabel-filled"
              data-shrink="true"
              for="dogs"
            >
              Dogs
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-marginDense MuiInput-marginDense"
              data-testid="select"
            >
              <div
                aria-haspopup="listbox"
                aria-labelledby="mui-component-select-dogs"
                class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input MuiInputBase-inputMarginDense MuiInput-inputMarginDense"
                id="mui-component-select-dogs"
                role="button"
                tabindex="0"
              >
                0
              </div>
              <input
                aria-hidden="true"
                class="MuiSelect-nativeInput"
                name="dogs"
                tabindex="-1"
                value="0"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSelect-icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root makeStyles-selector-4 MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            class="MuiFormControl-root MuiFormControl-marginDense MuiFormControl-fullWidth"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiFormLabel-filled"
              data-shrink="true"
              for="cats"
            >
              Cats
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-marginDense MuiInput-marginDense"
              data-testid="select"
            >
              <div
                aria-haspopup="listbox"
                aria-labelledby="mui-component-select-cats"
                class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input MuiInputBase-inputMarginDense MuiInput-inputMarginDense"
                id="mui-component-select-cats"
                role="button"
                tabindex="0"
              >
                0
              </div>
              <input
                aria-hidden="true"
                class="MuiSelect-nativeInput"
                name="cats"
                tabindex="-1"
                value="0"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSelect-icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root makeStyles-selector-4 MuiGrid-item MuiGrid-grid-xs-12"
        >
          <div
            class="MuiFormControl-root MuiFormControl-marginDense MuiFormControl-fullWidth"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-marginDense MuiFormLabel-filled"
              data-shrink="true"
              for="other"
            >
              Other
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-marginDense MuiInput-marginDense"
              data-testid="select"
            >
              <div
                aria-haspopup="listbox"
                aria-labelledby="mui-component-select-other"
                class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input MuiInputBase-inputMarginDense MuiInput-inputMarginDense"
                id="mui-component-select-other"
                role="button"
                tabindex="0"
              >
                0
              </div>
              <input
                aria-hidden="true"
                class="MuiSelect-nativeInput"
                name="other"
                tabindex="-1"
                value="0"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSelect-icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root makeStyles-error-6 MuiGrid-item MuiGrid-grid-xs-12"
      >
        <p
          class="MuiTypography-root makeStyles-body3-10 MuiTypography-body1"
        />
      </div>
      <div
        class="MuiGrid-root makeStyles-button-5 MuiGrid-item MuiGrid-grid-xs-12"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-containedSizeLarge MuiButton-sizeLarge MuiButton-fullWidth"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Next
          </span>
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
